lib/tasks/spree_mailchipm_ecommerce_tasks.rake

```diff
@@ -9,4 +9,18 @@ namespace :spree_mailchimp_ecommerce do
                                 currency_code: "USD"
                               })
   end
+
+  desc "Initial uploading of products list"
+  task upload_products: :environment do
+    logger = Logger.new("log/spree_mailchimp_initial_upload.log")
+    Spree::Product.find_in_batches do |batch|
```

Comment:
we can use `Spree::Product.find_each` in case we don't want to do anything in the scope of batches.